cypress/e2e/immutableDatabaseTests/linkCheck.ts

```diff
@@ -0,0 +1,116 @@
+import { setTokenUserViewPort } from '../../support/commands';
+
+// Formats paths to be displayed in an error log
+function formatPaths(paths: string[]): string {
+  let formattedPaths = '';
+  paths.forEach((path) => {
+    formattedPaths += '\n- ' + path;
+  });
+  return formattedPaths;
+}
+
+describe('Find broken anchor links', () => {
+  setTokenUserViewPort();
+
+  let visitedUrls: string[] = [
+    'https://twitter.com/DockstoreOrg', // Skip, test only fails because it redirects to login page if not logged in
```

Comment:
Bit of a band-aid solution, but I added the Twitter link here so that it doesn't cause the entire test to fail. It doesn't return the 200 status code because it redirects to Twitter's login page, but it's not really a "broken" link to be fixed.

Similar reason for adding http://localhost:4200/ to `visitedUrls` - it corresponds to the fake user's (nonexistent) avatar image in the navbar, which fails but doesn't actually affect the site's functionality/users.

Reply:
You could make it more explicit by creating a `skipUrls` array for URLs that are intentionally skipped (similar idea for images too).